cmd/minikube/cmd/console_test.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright (C) 2016 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"testing"
+	"os"
+	"io/ioutil"
+	"strings"
+)
+
+func TestDisplayConsoleInMachineReadable(t *testing.T) {
+	testDir, err := ioutil.TempDir("", "minishift-config-")
+	if err != nil {
+		t.Error()
+	}
+	defer os.RemoveAll(testDir)
+
+	f, err := os.Create(testDir + "out.txt")
```

Comment:
This works :-) Writing standard out to file and re-read. Even nicer imo is to use a Pipe. Then you need no additional file.